audio-analysis/videoHelper.py

```diff
@@ -107,7 +107,7 @@ def getAmplitudeAnalysis(self):
         # Test dictionary for graphing amplitudes
         raw_amplitudes = {}
 
-        fc, ic = 1, 1
+        fc, ic, fc_continuous = 1, 1, 1
```

Comment:
What does fc, ic, and fc_continuous mean?

Reply:
**fc** = frame count (this is reset after going through every interval)
_note that an interval can be 1, 5, or 10 seconds depending on audio time length_
**fc_continuous** = frame count continuous, same as above but not reset after every interval
**ic** = interval count, self explanatory.

Reply:
Ahh makes sense thanks 👍